homeassistant/components/devolo_home_network/strings.json

```diff
@@ -6,11 +6,17 @@
         "description": "[%key:common::config_flow::description::confirm_setup%]",
         "data": {
           "ip_address": "[%key:common::config_flow::data::ip%]"
+        },
+        "data_description": {
+          "ip_address": "IP address of the device that will be set up."
```

Comment:
Please check some others, we usually have it better described.

Bonus points if there's an easy way to discover the IP address from the app of the device so you can reference that.

(So for example "The IP address of the devolo hub. This can be found in settings -> something")